bindings/python/iota_sdk/types/output.py

```diff
@@ -123,6 +125,63 @@ class AccountOutput:
         init=False)
 
 
+@json
+@dataclass
+class AnchorOutput:
+    """Describes an anchor output.
+    Attributes:
+        amount :
+            The base coin amount of the output.
+        mana :
+            Amount of stored Mana held by this output.
+        unlock_conditions:
+            The conditions to unlock the output.
```

Comment:
Order!

Reply:
I didn't care much because we have an issue to order all python parameters, but I fixed it anyway